src/server/game/Cache/CharacterCache.h

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (C) 2008-2017 TrinityCore <http://www.trinitycore.org/>
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by the
+ * Free Software Foundation; either version 2 of the License, or (at your
+ * option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for
+ * more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#ifndef __CHARACTERCACHE_H
+#define __CHARACTERCACHE_H
+
+#include "Common.h"
```

Comment:
If you remove the typedefs from this file you can reduce the include set to just
"Define.h" (difference between Define and Common is huge right now)
"ObjectGuid.h"
<string>